numba/tests/test_withlifting.py

```diff
@@ -577,7 +597,7 @@ def foo(x):
         with self.assertRaises(errors.CompilerError) as raises:
             cfoo(x)
         self.assertIn(
-            ('unsupported controlflow due to return/raise statements inside '
+            ('unsupported controlflow due to raise statements inside '
```

Comment:
It's good that the difference can now be detected.

Reply:
it is, isn't it!